packages/api/cli/src/electron-forge-start.ts

```diff
@@ -2,15 +2,31 @@ import path from 'path';
 
 import { api, StartOptions } from '@electron-forge/core';
 import { ElectronProcess } from '@electron-forge/shared-types';
+import chalk from 'chalk';
 import program from 'commander';
 import fs from 'fs-extra';
+import semver from 'semver';
+// eslint-disable-next-line node/no-unpublished-import
+import updateNotifier from 'update-notifier';
 
 import './util/terminate';
 import workingDir from './util/working-dir';
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const metadata = require('../package.json');
 (async () => {
   let commandArgs = process.argv;
   let appArgs;
+  const notifier = updateNotifier({
+    pkg: {
+      name: metadata.name,
+      version: metadata.version,
+    },
+    updateCheckInterval: 1000 * 60 * 60,
+  });
+  if (notifier.update && semver.lt(notifier.update.current, notifier.update.latest)) {
```

Comment:
question: In what case is there going to be an update available and `current < latest`? I thought that `updateNotifier` would not actually report a `notifier.update` if there was not an update available.

Reply:
you are right 
`Returns an instance with an .update property if there is an available update, otherwise undefined.`
I was not thorough enough with the documentation and hence used semver to double check the condition. I'm sorry  :)